api/swagger-spec/openshift-openapi-spec.json

```diff
@@ -7,7 +7,7 @@
     "name": "Apache 2.0 (ASL2.0)",
     "url": "http://www.apache.org/licenses/LICENSE-2.0"
    },
-   "version": "v3.10.0+b3f250f-134-dirty"
```

Comment:
I've no idea why it keeps changing every time I run `./hack/update-generated-swagger-spec.sh` locally

Reply:
will CI pass if you don't change this?

Reply:
nope, it fails (which is good)